js/FocusManagementService.js

```diff
@@ -0,0 +1,40 @@
+class FocusManagementService {
+    /**
+     * Focus Management Service
+     */
+    constructor () {
+        this.$element = null;
+        this.focusableElementList = 'a[href], input, select, textarea, button';
+    }
+
+    /**
+     * Store element that focus will be returned to
+     * @param element {HTMLElement}
+     */
+    storeElement ($element) {
+        this.$element = $element;
+    }
```

Comment:
is `$element` here a `HTMLElement` or a jQuery collection? I suggest we reserve the dollar sign convention for jq objects only?

Reply:
It's a jQuery collection - I've updated the docblock